src/policy/policy.cpp

```diff
@@ -142,11 +143,16 @@ bool IsStandardTx(const CTransaction& tx, const std::optional<unsigned>& max_dat
             reason = "bare-multisig";
             return false;
         } else if (IsDust(txout, dust_relay_fee)) {
-            reason = "dust";
-            return false;
+            num_dust_outputs++;
         }
     }
 
+    // Only one dust is permitted(on otherwise valid ephemeral dust)
+    if (num_dust_outputs > 1) {
```

Comment:
if you make 1 a constexpr `MAX_DUST_OUTPUTS_PER_TX`, you can `static_assert` that it == 1 as an assumption in `CheckEphemeralSpends`

Reply:
great idea, added